ood-portal-generator/lib/ood_portal_generator/view.rb

```diff
@@ -19,7 +19,9 @@ def initialize(opts = {})
       @servername       = opts.fetch(:servername, nil)
       @proxy_server     = opts.fetch(:proxy_server, @servername)
       @port             = opts.fetch(:port, @ssl ? "443" : "80")
-      @logroot          = opts.fetch(:logroot, "logs")
```

Comment:
We should keep logroot for backwards compability and just use that value in the default value for accesslog and errorlog i.e.

    opts.fetch(:accesslog, "#{@logroot}/#{@servername ? @servername+'_access' : 'access'}#{@ssl ? '_ssl.log' : '.log'}")

Reply:
Fixed.